starboard/android/shared/test_filters.py

```diff
@@ -86,6 +86,18 @@
         # TODO: Filter this test on a per-device basis.
         'SbMediaCanPlayMimeAndKeySystem.MinimumSupport',
     ],
+    'blackbox': [
```

Comment:
There are existing filters for web platform tests in each platform's configuration.py—is there anything set up for black box test in the existing class there? I think we may want to use that file for now and keep web platform test filters side by side with black box test filters (but it would be great to completely merge all the test filters into a single file)

Reply:
See i.e. `starboard/linux/shared/cobalt/configuration.py`

Reply:
Yes, ideally to keep all the filtered tests at one place. Is test_filters.py good for this purpose?

Reply:
See the file above for filtered web platform tests